kibana/Dockerfile

```diff
@@ -6,7 +6,9 @@ ARG WAZUH_APP_VERSION="${WAZUH_VERSION}_${ELASTIC_VERSION}"
 
 USER root
 
-ADD  https://packages.wazuh.com/wazuhapp/wazuhapp-${WAZUH_APP_VERSION}.zip /tmp
+# App: 3.10.2 - 7.3.2 with this fix: https://github.com/wazuh/wazuh-kibana-app/issues/1815
+#ADD  https://packages.wazuh.com/wazuhapp/wazuhapp-${WAZUH_APP_VERSION}.zip /tmp
+COPY config/wazuhapp-${WAZUH_APP_VERSION}.zip /tmp
```

Comment:
```
Step 6/42 : COPY config/wazuhapp-${WAZUH_APP_VERSION}.zip /tmp
COPY failed: stat /var/lib/docker/tmp/docker-builder192295191/config/wazuhapp-3.10.2_7.3.2.zip: no such file or directory
```

This step and the following steps related to file **wazuhapp-${WAZUH_APP_VERSION}.zip** fail because the attachment in config has a different name, **wazuh-3.10.2_7.3.2.zip**. 

The file `kibana/config/wazuh-3.10.2_7.3.2.zip` should be renamed to `kibana/config/wazuhapp-3.10.2_7.3.2.zip`. 

Regards,

Alfonso Ruiz-Bravo